src/data/cards/cards.ts

```diff
@@ -0,0 +1,33 @@
+export interface DonationCardData {
+   id: number;
+   title: string;
+   text: string;
+   target: number;
+}
```

Comment:
I don't like```target``` as it is too vague, maybe ```donationTarget```, ```fundingGoal```?
And the same with instead of ```text``` maybe ```donationDescription```? Or even ```description```.